ui-core/src/components/inputs/Input.tsx

```diff
@@ -86,6 +86,7 @@ const Input = React.forwardRef<HTMLInputElement, InputProps>(
         disabled={disabled}
         required={required}
         small={small}
+        statusIconPosition="before-status-message"
```

Comment:
This is already in 'next-to-field' by default in `FieldWrapper.tsx`, we shouldn't add this